workshops/gloo-mesh-demo/install/http-client/http-client.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: http-client
   namespace: http-client
   labels:
-    istio-injection: enabled
+    istio.io/rev=1-15
```

Comment:
incorrect label

Reply:
Can you elaborate please? 1-15 does match the revision tag in the gm-istiod.yaml. Are you asking to revert it to 1-13 or make the revision tag 1-15-1 or 1-13-8?

Reply:
Oh never mind I see what you're talking about now. Should be:

   ``` istio.io/rev: 1-15```